packages/global/config/omeda-identity-x.js

```diff
@@ -60,4 +60,21 @@ module.exports = ({
   appendPromoCodeToHook,
   appendBehaviorToHook,
   appendDemographicToHook,
+  onLoginLinkSentFormatter: (async ({ req, payload }) => {
+    const omeda = req.app.locals.site.getAsObject('omeda');
+    const { source } = req.body;
```

Comment:
You can't rely on the `source` being present as a body param. Instead, you'll need to add the `source` to the emitted arguments to the onLoginLinkSent hook, in the [base-cms package](https://github.com/parameter1/base-cms/blob/master/packages/marko-web-identity-x/service.js#L220) and the [formatter](https://github.com/parameter1/base-cms/blob/master/packages/marko-web-omeda-identity-x/integration-hooks/on-login-link-sent.js#L13). Then it will be available for use in this function.